katago_server/trainings/models/network.py

```diff
@@ -11,10 +12,35 @@
 from katago_server.trainings.managers.network_pd_manager import NetworkPdManager
 from katago_server.trainings.managers.network_queryset import NetworkQuerySet
 
-
+# QUESTION (lightvector): Why does the server need to be uploading networks anywhere? Shouldn't it be the
+# case that simply networks are uploaded to the appropriate filestorage solution directly by training and
+# appropriately the server is notified?
```

Comment:
Nice question. In my mind, the  training will also do a multipart file data api call, same as npz files